app/javascript/components/ClientAlertProvider.tsx

```diff
@@ -29,22 +30,6 @@ export const ClientAlertProvider = ({ children }: { children: React.ReactNode })
     isVisible: false,
   });
 
-  const timerRef = React.useRef<number | null>(null);
```

Comment:
Why remove this?

Reply:
We don't need it anymore since the flash message is animated via a keyframe